docs/reference/installation.rst

```diff
@@ -68,211 +67,216 @@ Next, be sure to enable the ``Page`` and ``EasyExtends`` bundles in your applica
         );
     }
 
-Before we can go on with generating our Application files trough the `EasyExtendsBundle`_, we need to add some lines which we will override later (we need them now only for the following step):
-
-.. configuration-block::
-
-    .. code-block:: yaml
+Configuration
+-------------
 
-        # app/config/config.yml
+.. note::
+    If you are not using Symfony Flex, all configuration in this section should
+    be added to ``app/config/config.yml``.
 
-        sonata_page:
-            slugify_service:   sonata.core.slugify.cocur # old BC value is sonata.core.slugify.native
-            multisite:        host
-            default_template: default # template key from templates section, used as default for new pages
-            templates:
-                default:  { path: '@SonataPage/layout.html.twig',          name: 'default' }
-                2columns: { path: '@SonataPage/2columns_layout.html.twig', name: '2 columns layout' }
 
-            # Generates a snapshot when a page is saved (from the admin)
-            direct_publication: false # or %kernel.debug% if you want to publish in dev mode (but not in prod)
+Doctrine Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+Add these bundles in the config mapping definition (or enable `auto_mapping`_):
 
-Configuration
--------------
+.. code-block:: yaml
 
-To use the ``PageBundle``, add the following lines to your application
-configuration file.
+    # config/packages/doctrine.yaml
 
-.. note::
 
-    If your ``auto_mapping`` have a ``false`` value, add these lines to your
-    mapping configuration :
+    doctrine:
+        orm:
+            entity_managers:
+                default:
+                    mappings:
+                        ApplicationSonataPageBundle: ~ # only once the ApplicationSonataPageBundle is generated
+                        SonataPageBundle: ~
 
-.. configuration-block::
+        dbal:
+            types:
+                json: Sonata\Doctrine\Types\JsonType
```

Comment:
Never tried without it, in all docs we have this part. Will have to test later if it is possible not to use it and maybe deprecate that class

Reply:
Since this is a thing that should be applied to almost all sonata repositories using doctrine, can we discuss if this can be removed and replaced by the JsonType of doctrine in a separate Issue/PR? wdyt @greg0ire ?

Reply:
Sure

Reply:
Can this then get merged/approved and I will see to investigate later/tomorrow the json type